src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -53,6 +53,7 @@ const VaultChart = styled(Card)`
 
 const StyledCardContent = styled(CardContent)`
   margin-top: 0.4rem;
+  color: ${({ theme }) => theme.colors.texts};
```

Comment:
wrong color on classic:
Before:
![image](https://user-images.githubusercontent.com/78362532/167504486-27473376-33d0-4617-a377-193927544aac.png)


Now:
![image](https://user-images.githubusercontent.com/78362532/167504454-9f8d14a4-2bd6-4450-b369-a3a550fcaa2d.png)

Reply:
Looks better than before on cyberpunk tho!

Reply:
figma file specifies the darker grey - we have agreed with themes beyond light and dark we are ok to not comply with accessibility rules: 

<img width="1279" alt="image" src="https://user-images.githubusercontent.com/80323528/167602037-f6499976-d146-4095-8130-72fa4a4cce45.png">